ansible/vars/icds/icds_public.yml

```diff
@@ -175,7 +175,7 @@ etc_host_lines:
   - '10.247.24.13		www.icds-cas.gov.in'
 etc_host_lines_removed: ['etc_host_lines']
 
-http_proxy_address: '10.247.63.132'
+http_proxy_address: '10.247.24.16'
```

Comment:
that's right, i think in the meantime it's best that everybody uses es0 as proxy instead of dealing with a two-proxy configuration.